LICENSE.md

```diff
@@ -1,21 +1,10 @@
 MIT License
 
-Copyright (c) 2012  Chun-hao Hu Copyright (c) 2016-present Multiple Contributors
+Copyright (c) 2012 Chun-hao Hu
+Copyright (c) 2016-present Multiple Contributors
```

Comment:
I am not sure if multiple contributors should be put in more than one line for linters? What command as linter did you use?

Reply:
I only restored from _README_.
Anyway multiple copyrights must be placed on new line and maybe separated by empty line.
We can read [this](https://github.com/cs3org/.github).